### DISCUSSION TITLE
Query on Column/Attribute level lineage in Spline

### DISCUSSION THREAD

**Discussion by Sai7656** [AbsaOSS/spline-spark-agent#688] [Q&A]
HI Team,

I understand we can view the lineage by selecting the column in execution plan which is not much user-friendly if I want to look at the lineage for all columns(assume I have 200 columns) in my output dataset. I could see lineage tab in the dataset section as below which shows page under development and assume this will give lineage at dataset level in future release. Could someone clarify on this please.

![image](https://github.com/AbsaOSS/spline-spark-agent/assets/60571532/7f6cd534-fece-45f9-b78c-edc9a5f6a791)

**Comment by wajda** [comment]
Yes, the intention is to display the backward lineage and forward lineage (or impact) of the selected snapshot of the datasource. Unfortunately work on the project has slowed down significantly due to the lack of resources. The project book of work is undergoing review and reprioritisation at the moment, so hopefully you will see more activity on it in the near future.
